src/components/appmodallogin.main.jsx

```diff
@@ -162,6 +169,7 @@ class AppModalLoginMain extends React.Component {
                   {
                     (isLoading) ? <div className="miniLoader" /> : ('')
                   }
+                  <button type="button" className="label-link" onClick={this.resetPassword}>Forgot password?</button>
```

Comment:
This should only show up if the `forgot password` service is available. Please add a condition to surround this button here so it doesn't show up unless the API is available.